inst/report/Report_KRI.Rmd

```diff
@@ -74,17 +74,33 @@ if(setup$GroupLevel == "Site"){
 # subset data on latest snapshot date
 dfResults_current <- FilterByLatestSnapshotDate(params$dfResults)
 
-htmltools::div(
-  style = "table-layout: fixed; width: 100%; overflow: auto;",
-  Widget_GroupOverview(
-      dfResults = dfResults_current,
-      dfGroups = params$dfGroups,
-      dfMetrics = params$dfMetrics, 
-      strGroupLevel = setup$GroupLevel,
-      strGroupLabelKey = GroupLabelKey
-  )
+GroupOverViz <- Widget_GroupOverview(
+  dfResults = dfResults_current,
+  dfGroups = params$dfGroups,
+  dfMetrics = params$dfMetrics, 
+  strGroupLevel = setup$GroupLevel,
+  strGroupLabelKey = GroupLabelKey
 )
 
+# Remove sites whose metrics have no flags 
+dfResults_atleastoneflag <- params$dfResults %>% 
+  group_by(GroupID, MetricID) %>% 
+  mutate(flagsum = sum(Flag)) %>% # na.rm = TRUE depending on meaning
+  ungroup() %>% 
+  filter(flagsum != 0 | is.na(flagsum)) %>% 
+  select(-flagsum)
+
+
+FlagOverViz <- Widget_FlagOverTime(
+  dfResults = dfResults_atleastoneflag,
+  dfMetrics = params$dfMetrics,
+  strGroupLevel = setup$GroupLevel,
+  strFootnote = "*Only sites with at least one flag are shown."
+)
+
+overview_viz <- list(groupOverviewJS = GroupOverViz, flagOverTimeJS = FlagOverViz)
+
+Report_MetricCharts(overview_viz, overview = TRUE)
```

Comment:
Felt easier to make a small modification to `Report_MetricCharts()` that does the tabbing/knitting/printing than create a new function or hand write it into here? Certainly can modify